src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops_padded_gemm.cpp

```diff
@@ -36,11 +36,10 @@
 /// issues are not reproducible with ROCm no-npi build 6738-STG2 (proposed 4.2 release candidate).
 /// Let's disable these configs for 3.7...4.0.
 #define WORKAROUND_MI100_CONV_IMPLICIT_GEMM_HIP_FWD_V4R4_PADDED_GEMM_XDLOPS \
-    (HIP_PACKAGE_VERSION_FLAT >= 3007000000 && HIP_PACKAGE_VERSION_FLAT <= 4000999999)
+    (HIP_PACKAGE_VERSION_FLAT <= 4000999999)
```

Comment:
To be consistent with other code
```suggestion
    (HIP_PACKAGE_VERSION_FLAT <= 4000999999ULL)
```

Reply:
Right, thanks!

Reply:
Resolved